### DISCUSSION TITLE
[question] Is there any way to get nodeinfo informations from time to time

### DISCUSSION THREAD

**Discussion by gugagop** [zwave-js/zwave-js-ui#2667] [Q&A] [answered]
I need to get some nodes informations from time to time, but I didn't wanted to ask it every time. Is there any way to do it?

**Comment by robertsLando** [comment]
Using mqtt you mean? Why and what kind of info do you need?

**Comment by gugagop** [comment]
It is using mqtt. I need some info about the device. Battery for example.

**Comment by robertsLando** [comment, marked as answer]
Battery is a value, you should subscribe to that value changes
